core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,66 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
```

Comment:
> Can we add and order_id is null to this query though? That reduces the time for our data set on my dev box in postgres from ~60s to < 1s, since most of the data is good and we have an index on spree_adjustments.order_id.

I did that.